<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayManagerClass</class>
 <widget class="QMainWindow" name="DisplayManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="list_devices">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="list_inputs">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Display Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Input:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device List:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>10</y>
      <width>51</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolbar_top">
   <property name="windowTitle">
    <string>toolbar_top</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_refresh"/>
   <addaction name="action_savea"/>
   <addaction name="action_saveb"/>
   <addaction name="action_toggle"/>
   <addaction name="action_select"/>
   <addaction name="action_watch"/>
  </widget>
  <action name="action_refresh">
   <property name="text">
    <string>Refresh Devices</string>
   </property>
  </action>
  <action name="action_savea">
   <property name="text">
    <string>Save Local Profile</string>
   </property>
  </action>
  <action name="action_saveb">
   <property name="text">
    <string>Save Alt Profile</string>
   </property>
  </action>
  <action name="action_toggle">
   <property name="text">
    <string>ToggleProfiles</string>
   </property>
  </action>
  <action name="action_select">
   <property name="text">
    <string>Select HUB</string>
   </property>
  </action>
  <action name="action_watch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>watch HUB</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DisplayManager.qrc"/>
 </resources>
 <connections/>
</ui>
